lib/daemons/recurring_tasks.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require File.expand_path("base", File.dirname(__FILE__))
+
+Daemons::Base.new("recurring_tasks").start do
+  RecurringTaskConfig.recurring_tasks.each do |(worker, method)|
+    worker.new.public_send(method)
```

Comment:
It feels like there's room for an abstraction here. Maybe an internal `Task` class in our `RecurringTaskConfig` that handles the method chaining we're doing here? Our config could then implement `Enumerable`, and yield a series of tasks, which could let us simplify this daemon to something like `RecurringTaskConfig.each(&:invoke!)`.